src/extra/theme.light.css

```diff
@@ -19,3 +19,25 @@
     }
   }
 }
+
+[data-theme=light] {
```

Comment:
i feel like the data attribute styles should be opt-in, folks knowing their making a switch

Reply:
Yep. Shipping extra code should be opt-in. That said, I think you should probably _also_ include a 'all the bells and whistles' option you can import to get everything.